module/builder/consensus/config.go

```diff
@@ -3,28 +3,21 @@
 package consensus
 
 import (
-	"time"
+	"github.com/onflow/flow-go/consensus/hotstuff"
 )
 
 type Config struct {
-	minInterval time.Duration
-	maxInterval time.Duration
+	blockTimestamp hotstuff.BlockTimestamp
```

Comment:
same renaming suggestion: `BlockTimestamp` -> `Blocktimer`
```suggestion
	blocktimer hotstuff.Blocktimer
```